### DISCUSSION TITLE
Personal API options missing from My Account

### DISCUSSION THREAD

**Discussion by krakah** [dani-garcia/vaultwarden#2264] [Q&A]
Per the wiki for personal API which links to the bitwarden wiki:  https://bitwarden.com/help/personal-api-key/

When I navigate to My Account there is no API section.  

Version 2.25.0

**Comment by BlackDex** [comment]
There already is a PR for this in the web-vault patched version we use https://github.com/dani-garcia/bw_web_builds/pull/59
